app/models/tribont-shape-classificatie-code.ts

```diff
@@ -0,0 +1,10 @@
+import Model, { attr } from '@ember-data/model';
+
+declare module 'ember-data/types/registries/model' {
+  export default interface ModelRegistry {
+    'tribont-shape-classificatie-code': TribontShapeClassificatieCodeModel;
+  }
+}
+export default class TribontShapeClassificatieCodeModel extends Model {
```

Comment:
```suggestion
export default class TribontShapeClassificationCodeModel extends Model {
```

"classificatie" is dutch, this would require renaming it in the backend as well though.